### ISSUE TITLE
Possible to use a Scala 3 version of embedded-kafka?

### ISSUE THREAD

**Issue by bbarker** [zio/zio-kafka#1193]
When we try to use https://github.com/embeddedkafka/embedded-kafka-schema-registry in our project, we get a conflict:

```
[error] Modules were resolved with conflicting cross-version suffixes in ProjectRef(uri("file:/home/me/WH/xyz/"), "foo"):
[error]    io.github.embeddedkafka:embedded-kafka _2.13, _3
[error] Couldn't run bloopGenerate. The issue could be caused by failing to compile the project.
[warn] Removed stale /home/me/WH/xyz/.bloop/foo-test.json
```

It seems that zio-kafka is pulling in the 2.13 version of embedded kafka for the Scala 3 build; not sure if there's a way around this.

Here's how we're pulling in both deps:

```scala
lazy val zioKafkaTestKit          = zio                                   %% "zio-kafka-testkit"                   % "2.7.3"     % Test
lazy val embeddedkafkaSchemaReg   = "io.github.embeddedkafka"             %% "embedded-kafka-schema-registry"      % "7.6.0"     % Test
```

**Comment by erikvanoosten** [comment]
Duplicate of #1035.

With some time and love this problem might be solvable. More documentation about this is available on https://docs.scala-lang.org/scala3/guides/migration/compatibility-classpath.html

**Comment by bbarker** [comment]
Fortunately, for now, we have a workaround:

```scala
  lazy val embeddedkafkaSchemaReg = ("io.github.embeddedkafka" %% "embedded-kafka-schema-registry" % confluentKafkaV % Test)
    .cross(CrossVersion.for3Use2_13) exclude ("org.scala-lang.modules", "scala-collection-compat_2.13")
 ```
 
 Also posted in the other issue.

**Comment by calvinlfer** [comment]
Scala 3 without having to use CrossVersion

```scala
libraryDependencies += "io.github.embeddedkafka" %% "embedded-kafka"  % "3.8.0" % Test,
excludeDependencies += "org.scala-lang.modules" % "scala-collection-compat_2.13"
```

**Comment by erikvanoosten** [comment]
@calvinlfer If I understand correctly, if you use `zio-kafka-testkit`, you only need `excludeDependencies += "org.scala-lang.modules" % "scala-collection-compat_2.13"` because the testkit will already include embedded-kafka. Right?

I am asking because I want to add this to the documentation.

**Comment by erikvanoosten** [comment]
Ah right, we do exactly that already in zio-kafka itself. Okay, no worries.

**Comment by calvinlfer** [comment]
You are right 💯

**Comment by calvinlfer** [comment]
We can close this @erikvanoosten 🙏🏽

**Comment by erikvanoosten** [comment]
Solved by #1329.
